src/components/command-bar/commands/search/components/recent-searches/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { IconHistory16 } from '@hashicorp/flight-icons/svg-react/history-16'
+import { useCommandBar } from 'components/command-bar'
+import {
+	CommandBarButtonListItem,
+	CommandBarDivider,
+	CommandBarList,
+} from 'components/command-bar/components'
+import { RecentSearch, RecentSearchesProps } from './types'
+
+const RecentSearches = ({ recentSearches }: RecentSearchesProps) => {
+	const { inputRef, setCurrentInputValue } = useCommandBar()
+
+	if (!recentSearches || recentSearches.length === 0) {
+		return null
+	}
+
+	return (
+		<>
+			<CommandBarList label="Recent Searches">
+				{recentSearches.map((recentSearch: RecentSearch) => {
+					return (
+						<CommandBarButtonListItem
+							key={recentSearch}
+							icon={<IconHistory16 />}
+							onClick={() => {
+								setCurrentInputValue(recentSearch)
+								inputRef.current.focus()
+							}}
```

Comment:
future 💭 (no action needed): when we have multiple commands, do you think we would need to explicitly set the active command to "search" as part of this too?

Reply:
Hmmm I'm not sure. At the moment I would think not, because I don't think clearing the text input should have any effect on what command is active. I would expect that the user has to manually change the command themselves in some way.